src/metabase/pulse/render/style.clj

```diff
@@ -28,6 +28,10 @@
   "Used as color for 'We were unable to display this Pulse' messages."
   "#F9D45C")
 
+(def ^:const color-red
+  "Used as secondary color in charts"
+  "#EF8C8C")
```

Comment:
Got this from Kyle (it's `accent3` in the official colors guide).

Potentially a whitelabeling problem since the blue used as the primary color can be changed in the EE, but we don't let users set a secondary color.

Reply:
A few thoughts:

1. This is the exact same color as `color-error` below
1. Why not name it `accent-color` or something like that? That way if we decide we don't want it to be red anymore we don't need to go update the name of it anywhere. For example with `color-error` below, it's not called "color red" although it's actually the same color as this one. Better to name the constant with the purpose/use-case of the color than whatever it's current value is. 

      Another way to think about this: it would be like naming a constant `TEN_THOUSAND` instead of `CONNECTION_TIMEOUT_MS`

1.  If this is something that *should* be overridable for whitelabeling you could make it a function that always returns `#EF8C8C` right now and it would be easy enough to change the implementation in the future without having to change the usage... you should check with @kdoh and see if this is something we should be supporting. If it is, let's open another issue so we don't forget

Reply:
It's exposed in the EE admin UI as an "additional chart color", so I'm using that.